src/Microsoft.AspNetCore.ResponseCaching/CacheEntry/CacheEntryContext.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.ResponseCaching
+{
+    public class KeyValuePair
```

Comment:
Curious, why not use https://msdn.microsoft.com/en-us/library/5tbh8a42(v=vs.110).aspx